app/views/posts/index.html.erb

```diff
@@ -18,3 +18,7 @@
 <%= link_to new_post_path do %>
   New post
 <% end %>
+
+<%= link_to wall_index_url do %>
+  My Wall
```

Comment:
i'd like this to be
```ruby
 user_walls_path(user)
```
or something similar
http://guides.rubyonrails.org/routing.html#nested-resources